e2e/commands/start/start_test.go

```diff
@@ -380,4 +379,19 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf("The file or folder was ignored to send analysis:[%s]", testutil.NormalizePathToAssert(fileIgnored))))
 		})
 	})
+
+	When("--custom-path-rules is passed", func() {
+		customRulesJson := filepath.Join(os.TempDir(), fmt.Sprintf("horusec-custom-rules.json"))
```

Comment:
The `fmt.Sprintf` is unecessary.

Reply:
This file seems not exists. I think that would be better we create a custom rule file on temp dir and use it.